UI/ui_item.csv

```diff
@@ -9401,163 +9401,163 @@ MenuGuide#9,"""<c f0df60>Al comerciar con FUN, puede comprar varios objetos. Por
-Dark02_01#0,"""Técnica personalizada：<br>Técnica Oscura 2・Custom 1 Descripción<br>fila 3."""
-Dark03_01#0,"""Técnica personalizada：<br>Técnica Oscura 3・Custom 1 Descripción<br>fila 3."""
-Dark04_01#0,"""Técnica personalizada：<br>Técnica Oscura 4・Custom 1 Descripción<br>fila 3."""
-Dark05_01#0,"""Técnica personalizada：<br>Técnica Oscura 5・Custom 1 Descripción<br>fila 3."""
-Dark06_01#0,"""Técnica personalizada：<br>Técnica Oscura 6・Custom 1 Descripción<br>fila 3."""
-Dark07_01#0,"""Técnica personalizada：<br>Técnica Oscura 7・Custom 1 Descripción<br>fila 3."""
-Name_000#0,"""レアドロ倍率上昇"""
-Name_001#0,"""獲得経験値上昇"""
-Name_002#0,"""未鑑定品確率上昇"""
-Name_003#0,"""ＨＰ自動回復"""
-Name_004#0,"""回復薬効果上昇"""
-Name_005#0,"""確率物理耐性発生"""
-Name_006#0,"""確率法撃耐性発生"""
-Name_007#0,"""バーン無効化"""
-Name_008#0,"""フリーズ無効化"""
-Name_009#0,"""ショック無効化"""
-Name_010#0,"""ミラージュ無効化"""
-Name_011#0,"""パニック無効化"""
-Name_012#0,"""ポイズン無効化"""
-Name_013#0,"""パートナー強化"""
-Name_014#0,"""弱点ダメージアップ"""
+Wind06_01#0,"""Técnica personalizada<br>Técnica de viento 6・Custom 1 Descripción<br>fila 3."""
+Wind07_01#0,"""Técnica personalizada<br>Técnica de viento 7・Custom 1 Descripción<br>fila 3."""
+Light01_01#0,"""Técnica personalizada<br>Técnica de luz 1・Custom 1 Descripción<br>fila 3."""
+Light02_01#0,"""Técnica personalizada<br>Técnica de luz 2・Custom 1 Descripción<br>fila 3."""
+Light03_01#0,"""Técnica personalizada<br>Técnica de luz 3・Custom 1 Descripción<br>fila 3."""
+Light04_01#0,"""Técnica personalizada<br>Técnica de luz 4・Custom 1 Descripción<br>fila 3."""
+Light05_01#0,"""Técnica personalizada<br>Técnica de luz 5・Custom 1 Descripción<br>fila 3."""
+Light06_01#0,"""Técnica personalizada<br>Técnica de luz 6・Custom 1 Descripción<br>fila 3."""
+Light07_01#0,"""Técnica personalizada<br>Técnica de luz 7・Custom 1 Descripción<br>fila 3."""
+Dark01_01#0,"""Técnica personalizada<br>Técnica Oscura 1・Custom 1 Descripción<br>fila 3."""
+Dark02_01#0,"""Técnica personalizada<br>Técnica Oscura 2・Custom 1 Descripción<br>fila 3."""
+Dark03_01#0,"""Técnica personalizada<br>Técnica Oscura 3・Custom 1 Descripción<br>fila 3."""
+Dark04_01#0,"""Técnica personalizada<br>Técnica Oscura 4・Custom 1 Descripción<br>fila 3."""
+Dark05_01#0,"""Técnica personalizada<br>Técnica Oscura 5・Custom 1 Descripción<br>fila 3."""
+Dark06_01#0,"""Técnica personalizada<br>Técnica Oscura 6・Custom 1 Descripción<br>fila 3."""
+Dark07_01#0,"""Técnica personalizada<br>Técnica Oscura 7・Custom 1 Descripción<br>fila 3."""
+Name_000#0,"""Bono de rare drop """
+Name_001#0,"""Aumento de EXP """
+Name_002#0,"""Mayor probabilidad de objetos no identificados """
+Name_003#0,"""Restauracion automatica HP """
+Name_004#0,"""Bonificación en objeto curativo"""
+Name_005#0,"""Niega aleatoriamente el daño físico. """
+Name_006#0,"""Niega aleatoriamente el daño Tech."""
+Name_007#0,"""Previene estado quemar """
+Name_008#0,"""Previene la congelacion """
+Name_009#0,"""Previene Parálisis """
+Name_010#0,"""Previene Mirage """
+Name_011#0,"""Previene el Panico" ""
```

Comment:
You have a space between your ending quotes here.

Reply:
Thx